tensorhive/models/Reservation.py

```diff
@@ -151,16 +156,8 @@ def __repr__(self):
 
     starts_at={7}
     ends_at={8}
-    created_at={9}'''.format(
-            self.id, self.user_id,
-            self.title,
-            self.description,
-            self.protected_resource_id,
-            self.gpu_util_avg,
-            self.mem_util_avg,
-            self.starts_at,
-            self.ends_at,
-            self.created_at)
+    created_at={9}'''.format(self.id, self.user_id, self.title, self.description, self.protected_resource_id,
```

Comment:
Oversight + laziness. Repeating same words inside string and `format()` is driving me nuts.
But yes, it can be fixed. Please, leave a change suggestion here, so I can apply it from Github.